paramiko/client.py

```diff
@@ -311,6 +312,8 @@ def connect(
             for the SSH banner to be presented.
         :param float auth_timeout: an optional timeout (in seconds) to wait for
             an authentication response.
+        :param float channel_timeout: an optional timeout (in seconds) to wait
+             for a channel open response.
```

Comment:
@VakarisZ It looks like the aligment of "for a channel open response." is off by one space. Compare with the above and below parameters.

Reply:
```suggestion
        :param float channel_timeout: an optional timeout (in seconds) to wait
            for a channel open response.
```